src/types/user.type.ts

```diff
@@ -0,0 +1,10 @@
+export interface User {
```

Comment:
we have 3 types of user in the frontend as we need to have only one, check out the user type in src/api/types